src/steps/01-AcquisitionPackageDetails/COR_ACOR/ContactInfoForm.vue

```diff
@@ -88,6 +88,15 @@
         </v-col>
       </v-row>
 
+        <ATATPhoneInput
+          id="PhoneNumber"
+          label="Phone number"
+          class=" width-100"
+          :value.sync="_phone"
+          :country.sync="_selectedPhoneCountry"
+          :extension.sync="_phoneExt"
+        />
+
```

Comment:
- [ ] the prop `  @Prop({ default: true }) private phoneExtension!: boolean;` in the ATATPhoneInput.vue is not being used here. Is it necessary?